src/querypairmgr.h

```diff
@@ -21,8 +21,8 @@ struct hash_pair {
 };
 
 struct DnsTransaction {
-    timeval queryTS;
-    timeval totalTS;
+    timespec queryTS;
```

Comment:
I'm wondering about using `std::chrono::time_point` instead. Measurements of the durations and various conversion would come for free from the stdlib. But timespec is fine as well.

Reply:
Yes in early versions I using `std::chrono`, and I wound up switching to measuring with the packet time stamp instead of with the clock - because the former is available for free. One of the upstream changes was to switch from `timeval` to `timespec`, and this just follows that interface change.